docs/extensions/tutorials/_posts/1970-01-02-WritingATheme.md

```diff
@@ -375,3 +375,52 @@ Check `Customize theme` under the `Style` tab. You can see the `Restaurants` sec
 <p class="image">
 <img src='{{ site.url }}/img/tutorials/settings-theme/style-tab-themes-customise_theme-color_picker_preview.png'/>
 </p>
+
+## Adding a Custom Font
+
+In order to add a custom font to a theme, you'll have to add a `fonts` folder to your extension's `app` folder and add your custom fonts to it in `.TTF` format. You can see this structure in our [open sourced]("https://github.com/shoutem/extensions/tree/master/shoutem-rubicon-theme/app/fonts") theme extensions.
+
+These fonts also have to be linked into the app's binary using [`rnmp`](https://github.com/rnpm/rnpm). You do this by adding the following to your extension's `app/package.json` file:
+
+```JSON{5-9}
+#file: app/package.json
+{
+  "name": "{{site.example.devName}}.restaurants",
+  "version": "0.0.1",
+  "description": "My first extension.",
+  "rnpm": {
+    "assets": [
+      "fonts"
```

Comment:
Explain that it's `fonts` is the name of the folder. Explain or reference why to use `assets` field in `rnpm` field.